Tests/Specs/command-monitoring/tests/unified/redacted-commands.json

```diff
@@ -0,0 +1,659 @@
+{
+  "description": "redacted-commands",
+  "schemaVersion": "1.0",
```

Comment:
Is omitting such events definitely not spec compliant? It's not actually explicitly stated in the Command Monitoring spec (from my reading) that handshakes should be included in the events, though it also doesn't explicitly state that they shouldn't. IMO, given how easy it is to leak stuff and how noisy the handshake commands can be, I actually think it would be beneficial to mandate that drivers not emit events pre or during handshake. Rust is like libmongoc/Swift in this way btw.

Reply:
You're right, it's actually still ambiguous. I think I was thinking that it wasn't because of DRIVERS-1809, but we haven't actually done that ticket yet. FWIW that ticket originally proposed events should not be published, but after a conversation in leads triage - which I missed due to PTO so I don't have much context on - it was edited to propose the opposite.